templates/event.php

```diff
@@ -79,13 +79,30 @@
 		<h2><?php esc_html_e( 'Contributors', 'gp-translation-events' ); ?></h2>
 		<ul>
 			<?php foreach ( $contributors as $contributor ) : ?>
-			<li class="event-contributor" title="<?php echo esc_html( implode( ', ', $contributor->locales ) ); ?>"
+			<li class="event-contributor" title="<?php echo esc_html( implode( ', ', $contributor->locales ) ); ?>">
```

Comment:
Nice bugfix :) I'm surprised it was working correctly due to the HTML being malformed.